packages/block-editor/src/components/block-list/block.js

```diff
@@ -775,5 +775,6 @@ export default compose(
 	withViewportMatch( { isLargeViewport: 'medium' } ),
 	applyWithSelect,
 	applyWithDispatch,
+	ifCondition( ( { block } ) => !! block ),
```

Comment:
I think we want a comment block here explaining why this was added.

Reply:
indeed, I referenced the issue and briefly explained why the block might not be defined